### DISCUSSION TITLE
Create a fixed weekly work schedule?

### DISCUSSION THREAD

**Discussion by gd-vanloi** [nhn/tui.calendar#894] [Q&A] [answered]
Hi, I need someone to help me how to create a fixed teaching schedule for weekdays, as shown below.

![Screen Shot 2021-09-10 at 09 26 55](https://user-images.githubusercontent.com/15335651/132789433-d81ce5d0-e6ad-4250-a808-df47cff19a8a.png)

It can be considered as your fixed work schedule.

**Comment by adhrinae** [comment, marked as answer]
@gd-vanloi 
Look like you're looking for the fixed 'recurring event' functionality.
Unfortunately, `tui.calendar` doesn't have that kind of feature.
I might suggest creating fixed schedules by dates every time the calendar is rendered.

However, If you only need to implement a timetable, you can leverage the [tui.grid](https://github.com/nhn/tui.grid) for that.
